__tests__/app.test.js

```diff
@@ -234,12 +238,144 @@ xdescribe('/api/articles', () => {
 
         })
 
+        test('/api/articles?comment_count responds 200 with array of objects and extra property count_coments ', () => {
+            return request(app).get('/api/articles?comment_count').expect(200).then((response) => {
```

Comment:
Do bear in mind that this query is incomplete - we will always want to apply a key/value pair rather than just the key.
As mentioned in a prior PR, the articles should be served up with a comment_count by default